repos/system_upgrade/el7toel8/actors/cupscheck/actor.py

```diff
@@ -0,0 +1,226 @@
+from leapp.actors import Actor
+from leapp.libraries.common.reporting import report_generic
+from leapp.models import Report, InstalledRedHatSignedRPM, Cupsmigrationmodel, Matrix
+from leapp.tags import ChecksPhaseTag, IPUWorkflowTag
+
+import os
+from re import compile, findall, MULTILINE
+
+"""
+CUPS configuration files
+"""
+CUPSD_CONF = '/etc/cups/cupsd.conf'
+CUPSFILES_CONF = '/etc/cups/cups-files.conf'
+
+"""
+Names for better navigation through change list
+"""
+KEY = 0
+DIRECTIVES = 1
+SOURCE = 2
+TARGET = 3
+
+"""
+List of changed directives
+
+Scheme:
+Dictionary key # Directives # Input configuration file # Output configuration file
+
+Some changes happened only in one configuration file, so the last member of list is 'None'.
+"""
+cupsd_changes_list = [
```

Comment:
As mentioned on other file, I have the feeling that this can be better represented using a dictionary or namedtuples.

Reply:
Used different approach, hopefully more understandable.